### DISCUSSION TITLE
m3u8 generator

### DISCUSSION THREAD

**Discussion by work-shiju** [bluenviron/mediamtx#3465] [Questions]
### Question

hi am using mediamtx for record cctv feeds .i can able to get .ts file but i cant able to find .m3u8 file can u pls suggest something for creating m3u8

**Comment by moenodedev** [comment]
MediaMTX does not generate m3u8 playlists for recordings. #3243 has one approach for creating them.

**Reply by work-shiju** [reply]
thank u for u r response, i have one more query is there is a chance to relay rtsp stream to some other server using mediamtx

**Reply by moenodedev** [reply]
By push or pull? If the other server can ingest HLS the URL is http://yourserver:8888/yourpath/index.m3u8

https://github.com/bluenviron/mediamtx?tab=readme-ov-file#hls

**Comment by aler9** [comment]
Hello, the only way we support to serve recordings consists in using the [Playback server](https://github.com/bluenviron/mediamtx?tab=readme-ov-file#playback-recorded-streams), which provides URLs that are natively compatible with the \</video> tag, any browser and any player, without the need for a m3u8 file.

[a comment by an automated account is not shown]
